src/main/java/org/roda_project/commons_ip2/validator/common/ZipManager.java

```diff
@@ -48,7 +48,7 @@ public InputStream getMetsRootInputStream(Path path) throws IOException {
     String entry = null;
     while (entries.hasMoreElements()) {
       ZipEntry entr = (ZipEntry) entries.nextElement();
-      if (entr.getName().matches(".*/METS.xml")) {
+      if (entr.getName().endsWith("METS.xml")) {
```

Comment:
Note that this code will consider any file ending with "METS.xml" as entering this if-branch. For example, a file named "notTheMETS.xml" will be considered.

```suggestion
      if (entr.getName().endsWith("/METS.xml")) {
```